File: Practice/13/Python/simple number.py
while True:
    num = int(input('Ведите число\n'))
    if num >= 2:
        for cnt in range(2, int(num**0.5)+1):
            if num % cnt == 0:
                print('Составное')
                break
        else:
            print('Простое')
        break
    else:
        print ('Вы ввели некорректные данные. Попробуйте ещё раз\n')

File: Practice/13/C++/simple number/simple number/simple number.cpp
#include <iostream>
using namespace std;

int main()
{
	setlocale(LC_ALL, "Russian");

	long num;
	int cnt, i = 0;
	cout << "Введите число\n";
	cin >> num;
	if (num >= 2)
	{
		for (cnt = 2; cnt <= floor(sqrt(num)); cnt++)
		{
			if (num % cnt == 0)
				i++;
		}
		if (i > 0)
			cout << "Составное";
		else
			cout << "Простое";
	}
	else
	{
		cout << "Вы ввели некорректные данные. Попробуйте ещё раз\n\n";
		return main();
	}
}